source/jormungandr/jormungandr/realtime_schedule/siri.py

```diff
@@ -91,11 +91,16 @@ class Siri(RealtimeProxy):
         </x:Body>
     </x:Envelope>'
 
-    Then Navitia matches route-points in the response using {stop_code}, {route_code} and {line_code}.
-
     {datetime} is iso-formated: YYYY-mm-ddTHH:MM:ss.sss+HH:MM
-    {stop_code}, {route_code} and {line_code} are provided using the same code key, named after
-    the 'destination_id_tag' if provided on connector's init, or the 'id' otherwise.
+    {requestor_ref} is a configuration parameter
+    {stop_code} is the stop_point code value, which type is the id of the connector
```

Comment:
As it is the same mechanism as response matching, maybe this add:
```suggestion
    {stop_code} is the stop_point code value, which type is the 'id' of the connector (or 'destination_id_tag' if provided in conf)
```